pumps/prometheus.go

```diff
@@ -67,6 +67,12 @@ type PrometheusMetric struct {
 	MetricType string `json:"metric_type" mapstructure:"metric_type"`
 	// Defines the buckets into which observations are counted. The type is float64 array and by default, [1, 2, 5, 7, 10, 15, 20, 25, 30, 40, 50, 60, 70, 80, 90, 100, 200, 300, 400, 500, 1000, 2000, 5000, 10000, 30000, 60000]
 	Buckets []float64 `json:"buckets" mapstructure:"buckets"`
+	// Controls whether the pump client should hide the API key. In case you still need substring
+	// of the value, check the next option. Default value is `false`.
+	ObfuscateAPIKeys bool `json:"obfuscate_api_keys" mapstructure:"obfuscate_api_keys"`
+	// Define the number of the characters from the end of the API key. The `obfuscate_api_keys`
+	// should be set to `true`. Default value is `4`.
+	ObfuscateAPIKeysLength int `json:"obfuscate_api_keys_length" mapstructure:"obfuscate_api_keys_length"`
```

Comment:
I think we can remove this config, if we can calculate how many digits to display/hide would be the best, or, we can just set arbitrary number of digits in the same way as we do in the gateway, check: https://github.com/TykTechnologies/tyk/blob/master/gateway/log_helpers.go#L14

what do you think? is there any special case for which you think we should keep it as a config?